cameraconnect/src/trigger.py

```diff
@@ -182,20 +190,31 @@ def __init__(self,cam,interface,cycle_time) -> None:
         while True:
             # Get actual time and save it as start time
             timer_start = time.time()
-            
-            self.cam.get_image()
+            while cycle_time > abs(time.time() - timer_start):
+                try:
+                    self.cam.get_image()
+                    break
+                except Exception as _e:
+                    ttw = cycle_time * 0.1
+                    logging.critical(f"Failed to get image at:{datetime.datetime.now(tz=datetime.timezone.utc)} "
```

Comment:
should not this be a normal error? critical usually refers to an error that cannot be recovered (https://www.techopedia.com/definition/6667/critical-error)

Reply:
reduced log level to ERROR